extensions/default/src/DicomWebDataSource/utils/clientManager.ts

```diff
@@ -0,0 +1,215 @@
+import { api } from 'dicomweb-client';
+import StaticWadoClient from './StaticWadoClient';
+import dcm4cheeReject from '../dcm4cheeReject';
+import { errorHandler, utils } from '@ohif/core';
+
+export default class ClientManager {
+  clients;
+  userAuthenticationService;
+
+  constructor({ params, query, dicomWebConfig, userAuthenticationService }) {
+    this.clients = [];
+    this.userAuthenticationService = userAuthenticationService;
+    if (Array.isArray(dicomWebConfig)) {
+      dicomWebConfig.forEach(config => this.addConfiguration(params, query, config));
+    } else {
+      this.addConfiguration(params, query, dicomWebConfig);
+    }
+  }
+
+  /**
+   * Adds a dicomweb server configuration in the clients list
+   * @param configToAdd
+   * @returns {void}
+   */
+  private _addConfiguration(configToAdd): void {
```

Comment:
I believe the common practice is to put the public methods first, then private methods. 
@igoroctaviano wdyt?

Reply:
That's right, and if you are using private you don't need underscore. [Also its not a very good practice](https://google.github.io/styleguide/tsguide.html).

Reply:
I still see some private methos above the public ones.